cudamapper/src/main.cu

```diff
@@ -242,9 +252,7 @@ int main(int argc, char* argv[])
     std::shared_ptr<claragenomics::DeviceAllocator> allocator(new claragenomics::CudaMallocAllocator());
 #endif
 
-    auto compute_overlaps = [&](const query_target_range query_target_range, const int device_id) {
-        std::vector<std::shared_ptr<std::future<void>>> print_pafs_futures;
-
+    auto compute_overlaps = [&](const QueryTargetRange query_target_range, const int device_id) {
```

Comment:
Can `query_target_range` be passed by reference?

Reply:
fixed

Reply:
yes. fixed